bento_beacon/config_files/config.py

```diff
@@ -38,10 +38,15 @@ class Config:
 
 # -------------------
 # handle injected config files
-
+#   a) obtain reference to the expected configuration files' location by
+#      using the programmable env variable `CONFIG_ABSOLUTE_PATH` if it exists, or
+#   b) default to using "this file's directory" as the reference to where
+#      configuration files are expected to be located
     def retrieve_config_json(filename):
-        config_path = os.path.dirname(os.path.abspath(__file__))
-        file_path = os.path.join(config_path, filename)
+        # TODO: abstract out CONFIG_PATH if needed
+        CONFIG_PATH = os.environ.get("CONFIG_ABSOLUTE_PATH", os.path.dirname(os.path.abspath(__file__)))
```

Comment:
generally python convention is variables are uppercase only if they're global, so not if they're declared in a function - the environment variable change is good though